sdks/python/apache_beam/runners/dask/dask_runner.py

```diff
@@ -159,6 +163,8 @@ def is_fnapi_compatible():
     return False
 
   def run_pipeline(self, pipeline, options):
+    import dask
```

Comment:
The try..except is a recent addition (to deal with a test-time-only threading issue). Nested imports are what I've had for a long time. That's the main reason for why things are the way they are. 

Nested imports are there so that users don't have to depend on dask to load the runner.